### PR TITLE
Docs and nulls

### PR DESCRIPTION
**Pull request by ojomio**
Added docs and fixed null order

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pydantic-avro"
-version = "0.3.2"
+version = "0.4.0"
```
Comment (ffinfo) [inline pyproject.toml:3]: can you please revert this version change?
Will do this is a release

Comment (ojomio) [reply pyproject.toml:3]: sure

File: src/pydantic_avro/avro_to_pydantic.py [hunk 2]
```diff
@@ -72,22 +72,22 @@ def get_python_type(t: Union[str, dict]) -> str:
...
         current = f"class {name}(BaseModel):\n"
-
+        missing = object()
         for field in schema["fields"]:
             n = field["name"]
             t = get_python_type(field["type"])
-            default = field.get("default")
-            if default is None:
+            default = field.get("default", missing)
+            if default is missing:
```
[the remark this answers is not in the document]
Comment (ojomio) [reply src/pydantic_avro/avro_to_pydantic.py]: Yeah, it makes sense

**Review by ffinfo — APPROVED** [review]

**Comment by ffinfo** [comment]
@ojomio was this this ready? for me this looks fine :)

**Comment by ojomio** [comment]
Sorry, was on vacations) Seems to be ready, will provide an update today

File: src/pydantic_avro/base.py [hunk 3]
```diff
@@ -40,6 +40,10 @@ def get_type(value: dict) -> dict:
             avro_type_dict: Dict[str, Any] = {}
             if "default" in value:
                 avro_type_dict["default"] = value.get("default")
+            if "description" in value:
+                avro_type_dict["doc"] = value.get("description")
+            if "allOf" in value and len(value["allOf"]) == 1:
```
Comment (ojomio) [inline src/pydantic_avro/base.py:45]: This was added for enums, frankly speaking, I do not know if cases with more allOf constraints arise, but this particular workaroud helps to handle the common case

**Comment by ffinfo** [comment]
version 0.4.1 will include this

**Comment by ojomio** [comment]
Thank you for support!

**Pull request merged**